.evergreen/auth_aws/setup_secrets.py

```diff
@@ -6,19 +6,34 @@
 import json
 import os
 import sys
+import uuid
 
 import boto3
 import botocore.exceptions
 
+AWS_ROLE_ARN = "arn:aws:iam::857654397073:role/drivers-test-secrets-role"
```

Comment:
Could we set this in an environment variable? Not a huge fan of setting even non-secret credentials inside a script.

Reply:
There are other places we hard-code ARNs or links in this repo.  The intent here was that it should be entirely self-contained without drivers needing to manually update EVG project config, which is part of the reason for having this script in general.

Reply:
I'd prefer we migrate away form hard-coding ARNs in general, but that's a different discussion entirely. In the context of the existing repo, makes sense.